starter-code/french-onion-soup-starter-code/css/main.css

```diff
@@ -1,9 +1,105 @@
-/*Some necessary adjustments*/
-h1, body, html, main {
-  margin: 0;
-}
-/*Import the font and set the default size*/
-html {
-  font-family: 'Open Sans', sans-serif;
-  font-size: 18px
-}
+/*Some necessary adjustments*/
+h1, body, html, main {
+  margin: 0;
+}
+/*Import the font and set the default size*/
+html {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px
+}
+body{
+	font-family:Arial, helvetica
+}
+
+#pagewrap {
+	background-color: #fff;
+     width: 60%;
+     margin: 0 auto;
+}
+#contentnoapple-wrap{
+	/* background-color:red; */
+     width: 80%;
+     margin: 0 auto;
+}
+/* #imapp{
+	background-image: url("images/apple-pie2.jpg");
+	height:400px;
+	width:900px;
+	background-repeat: no-repeat;
+} */
+#applebigpicture{
+	display:block;
+	max-width:100%;
+	height:auto;
+	margin: 0 auto;
+}
+#he1{
+	
+	position:relative;
+	padding:0;
+	margin-top:-300px;
+	margin-bottom:300px;
+	text-align:center;
+	font-size:4em;
+	color:white;
+	font-weight:100;
+	letter-spacing: 1.5px;
+}
+#parragrafdiv{
+	margin: 5px;
+	/* background-color:red; */
+	padding:10px;
+}
+#intro{
+	font-size: 1.3em;
+	font-weight:100;
+	color:#464637;
+}
+h2{
+	color:#464637;
+	font-weight:lighter
+	
+}
+.line{
+	border-bottom: 2px solid #cccccc;
+	padding-bottom:20px
+}
+#cookinfo{
+	height:60px;
+}
+.ingr-direc{
+	font-size:1em
+	
+}
+
+ul{
+	padding-left:15px;
+	list-style-type:none;
+	line-height: 1.5;
+	font-weight:100;
+	color:#464637;
+}
+ol{

```

Comment:
Be careful with tag selectors.  In a bigger project, you'll see why.  But it's fine for this assignment, just an FYI for the future.